code/modules/admin/admin_verbs.dm

```diff
@@ -1061,12 +1061,39 @@ var/list/admin_verbs_cciaa = list(
 	set desc = "Tells everyone to man up and deal with it."
 
 	for (var/mob/T as mob in mob_list)
-		to_chat(T, "<br><center><span class='notice'><b><font size=4>Man up.<br> Deal with it.</font></b><br>Move on.</span></center><br>")
-		sound_to(T, 'sound/voice/ManUp1.ogg')
+		T << "<br><center><span class='notice'><b><font size=4>Man up.<br> Deal with it.</font></b><br>Move on.</span></center><br>"

```

Comment:
okay basically revert all of the to_chat -> X << Y changes in this file please.

Reply:
yeah its pretty simple i just merged badly